airbyte-integrations/connectors/source-dv-360/acceptance-test-config.yml

```diff
@@ -0,0 +1,37 @@
+# See [Source Acceptance Tests](https://docs.airbyte.io/connector-development/testing-connectors/source-acceptance-tests-reference)
+# for more information about how to configure these tests
+connector_image: airbyte/source-dv-360:dev
+tests:
+  spec:
+    - spec_path: "source_dv_360/spec.json"
+  connection:
+    - config_path: "secrets/config.json"
+      status: "succeed"
+    - config_path: "integration_tests/invalid_config.json"
+      status: "failed"
+  discovery:
+    - config_path: "secrets/config.json"
+  basic_read:
+    - config_path: "secrets/config.json"
+      configured_catalog_path: "integration_tests/configured_catalog.json"
+      empty_streams:
+        [
+          "standard",
+          "reach",
+          "floodlight",
+          "unique_reach_audience",
+          "audience_composition",
+        ]
```

Comment:
doesnt look right all this empty streams here if you're implementing a new connector.

Reply:
also most of them doens't show in the configured catalog.

Reply:
To be honest I wasn't sure about what to put here, I just looked at what was done in other connectors.